sklearn/tests/test_multiclass.py

```diff
@@ -470,16 +482,28 @@ def test_ovo_partial_fit_predict():
     assert_almost_equal(pred1, pred2)
 
     # Test when mini-batches don't have all target classes
-    ovo1 = OneVsOneClassifier(MultinomialNB())
-    ovo1.partial_fit(iris.data[:60], iris.target[:60], np.unique(iris.target))
-    ovo1.partial_fit(iris.data[60:], iris.target[60:])
```

Comment:
@MechCoder [perm](https://github.com/kaichogami/scikit-learn/blob/partial_fit_fix/sklearn/tests/test_multiclass.py#L37) mixes up the indexes of `data` and `target classes`. Previously I failed to notice this, that is why the tests passed in #6087. We could perhaps remove the `perm` function call, if it doesn't have any other affect anywhere else.